aries-test-harness/features/0160-connection.feature

```diff
@@ -68,7 +68,7 @@ Feature: RFC 0160 Aries agent connection functions
                         # the agents under test implement acks.
 
 
-   @T003-AIP10-RFC0160 @SingleUseInvite @P2 @normal @ExceptionTest @WillFail @OutstandingBug..418..https://github.com/hyperledger/aries-cloudagent-python/issues/418 @allure.issue:https://github.com/hyperledger/aries-cloudagent-python/issues/418 @RFC0160
+   @T003-RFC0160 @SingleUseInvite @AIP10 @normal @ExceptionTest @WillFail @OutstandingBug..418..https://github.com/hyperledger/aries-cloudagent-python/issues/418 @allure.issue:https://github.com/hyperledger/aries-cloudagent-python/issues/418 @RFC0160
```

Comment:
Randomly picking this one --- shouldn't there be an @RFC0160 tag on this?  Or is this the inheritance experiment?

Reply:
It is. It's at the end of the line.

Reply:
Got it -- thanks. Not crucial, but maybe have all the RFCs are the start -- maybe an order like this: 

<wip - if present> <ID> <AIP(s)> <RFC(s)> <other> <issue(s)>

Reply:
Note that, if the tag inheritance works with allure, then the order, or where these tags are, will be taken care of by Behave/Allure. AIP and RFC tags will go up and tag at the feature level, and the tests will inherit those tags.

Reply:
Tags reordered.